microsoft/promptflow
# Refine sdk resume test

# Description

Refine sdk_cli_test and sdk_cli_azure_test:

- Add test case for llm and check the token num is valid.
- Add test case for image and aggregation node.
- Add more assertion, such as success result of original run, new result of resume run and so on.

# All Promptflow Contribution checklist:
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices

### Testing Guidelines
